Netflix/Screens/Dashboard/Home/HomeViewController.swift

```diff
@@ -6,10 +6,183 @@
 
 class HomeViewController: UIViewController {
-
+    
+    private let latestFilmImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: Asset.Assets.filmCover.name)
+        return imageView
+    }()
+    
+    private let latestFilmTitle: UILabel = {
+        let label = UILabel()
+        label.text = "Never Have I Ever"
+        label.backgroundColor = .clear
+        label.textAlignment = .center
+        label.textColor = .white
+        label.font = UIFont(name: "Mansalva-Regular", size: 48)
+        label.adjustsFontForContentSizeCategory = true
+        label.numberOfLines = 2
+        return label
+    }()
+    
+    private let netflixButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(named: Asset.Assets.logoNetflix.name), for: .normal)
+        button.imageView?.snp.makeConstraints({ make in
+            make.width.height.equalToSuperview()
+        })
+        return button
+    }()
+    
+    private let profileButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(systemName: "person.fill"), for: .normal)
+        button.imageView?.snp.makeConstraints({ make in
+            make.width.height.equalToSuperview()
+        })
+        button.tintColor = .white
+        return button
+    }()
+    
+    private let likeButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(systemName: "heart"), for: .normal)
+        button.setTitle("Like", for: .normal)
+        button.tintColor = .white
+        return button
+    }()
+    
+    private let playButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(systemName: "play.fill"), for: .normal)
+        button.setTitle("Play", for: .normal)
+        button.setTitleColor(.black, for: .normal)
+        button.tintColor = .black
+        button.backgroundColor = .white
+        button.contentMode = .scaleAspectFit
+        return button
+    }()
+    
+    private let popularMoviesLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Popular Movies"
+        label.backgroundColor = .clear
+        label.textAlignment = .left
+        label.textColor = .white
+        label.font = UIFont.preferredFont(forTextStyle: .title3)
+        label.adjustsFontForContentSizeCategory = true
+        return label
+    }()
+    
+    private let popularMoviesCollection: UICollectionView = {
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .horizontal
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collectionView.register(HomeCollectionViewCell.self, forCellWithReuseIdentifier: HomeCollectionViewCell.identifier)
+        return collectionView
+    }()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        popularMoviesCollection.delegate = self
+        popularMoviesCollection.dataSource = self
```

Comment:
To keep all collectionRelated staff in one place
```suggestion
    private lazy var popularMoviesCollection: UICollectionView = {
        let layout = UICollectionViewFlowLayout()
        layout.scrollDirection = .horizontal
        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
         popularMoviesCollection.delegate = self
        popularMoviesCollection.dataSource = self
        collectionView.register(HomeCollectionViewCell.self, forCellWithReuseIdentifier: HomeCollectionViewCell.identifier)
        return collectionView
    }()
    
    override func viewDidLoad() {
        super.viewDidLoad()
```